modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -235,15 +235,19 @@ public String toDefaultValue(Schema p) {
             // convert the example if it exists
             Object currentExample = p.getExample();
             if (currentExample != null) {
-                date = (Date) currentExample;
+                try {
+                    date = (OffsetDateTime) currentExample;
+                } catch (ClassCastException e) {
+                    date = ((Date) currentExample).toInstant().atOffset(ZoneOffset.UTC);
```

Comment:
This second operation should probably be wrapped in a try/catch with a warning which returns an empty string whenever the value can't be parsed.

Reply:
I have added a warning.